inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/utils/file/NewDateUtils.java

```diff
@@ -207,15 +207,11 @@ private static Calendar getDateTime(Calendar calendar, String cycleUnit, String
     public static boolean isValidCreationTime(String dataTime, String cycleUnit,
             String timeOffset) {
         long timeInterval = 0;
-        if ("Y".equalsIgnoreCase(cycleUnit)) {
+        if (CycleUnitType.DAY.equalsIgnoreCase(cycleUnit)) {
             timeInterval = DAY_TIMEOUT_INTERVAL;
-        } else if ("M".equals(cycleUnit)) {
+        } else if (CycleUnitType.HOUR.equalsIgnoreCase(cycleUnit)) {
             timeInterval = HOUR_TIMEOUT_INTERVAL;
-        } else if ("D".equalsIgnoreCase(cycleUnit)) {
-            timeInterval = DAY_TIMEOUT_INTERVAL;
-        } else if ("h".equalsIgnoreCase(cycleUnit)) {
-            timeInterval = HOUR_TIMEOUT_INTERVAL;
-        } else if (cycleUnit.contains("m")) {
+        } else if (cycleUnit.contains(CycleUnitType.MINUTE)) {
```

Comment:
why `cycleUnit.contains` here?

Reply:
Due to historical reasons, there were originally situations of 10m and 30m

Reply:
so may be `endwith` is better?

Reply:
ok